project_config.yml

```
@@ -0,0 +1,3 @@
+dataset_location: /Volumes/dbw-dplabdlz-dev-uks-01/chris_training/sample_data/twitter_sentiment.csv
+train_pc: 0.8
+test_pc: 0.2
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)